niworkflows/viz/utils.py

```diff
@@ -403,8 +436,8 @@ def compose_view(bg_svgs, fg_svgs, ref=0, out_file='report.svg'):
 
 def _which(cmd):
     try:
-        subprocess.run([cmd], stdin=subprocess.DEVNULL,
-                       stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
+        subprocess.run([cmd], input=subprocess.DEVNULL, stdout=subprocess.DEVNULL,
```

Comment:
`input` takes a `bytes` and `stdin` takes a filehandle/`subprocess.DEVNULL`. I'd just go with:

```Python
input=b''
```